utilities/pipelines/resourcePublish/Publish-ModuleToBicepRegistry.ps1

```diff
@@ -76,16 +79,24 @@ function Publish-ModuleToPrivateBicepRegistry {
             }
         }
 
+        if ($BicepRegistryPublic) {
+            if ($PSCmdlet.ShouldProcess("Container Registry [$BicepRegistryName] public (anonymous) access", 'Enable')) {
+                az acr update --name $BicepRegistryName --anonymous-pull-enabled
+            }
+        }
+
         # Extracts Microsoft.KeyVault/vaults from e.g. C:\modules\Microsoft.KeyVault\vaults\deploy.bicep
         $moduleIdentifier = (Split-Path $TemplateFilePath -Parent).Replace('\', '/').Split('/modules/')[1]
         $moduleRegistryIdentifier = 'bicep/modules/{0}' -f $moduleIdentifier.Replace('\', '/').Replace('/', '.').ToLower()
 
         #############################################
-        ##    Publish to private bicep registry    ##
+        ##    Publish to bicep registry    ##
         #############################################
         $publishingTarget = 'br:{0}.azurecr.io/{1}:{2}' -f $BicepRegistryName, $moduleRegistryIdentifier, $ModuleVersion
-        if ($PSCmdlet.ShouldProcess("Private bicep registry entry [$moduleRegistryIdentifier] version [$ModuleVersion] to registry [$BicepRegistryName]", 'Publish')) {
+        $publishingTargetLatest = 'br:{0}.azurecr.io/{1}:{2}' -f $BicepRegistryName, $moduleRegistryIdentifier, 'latest'
+        if ($PSCmdlet.ShouldProcess("Bicep registry entry [$moduleRegistryIdentifier] version [$ModuleVersion] to registry [$BicepRegistryName]", 'Publish')) {
             bicep publish $TemplateFilePath --target $publishingTarget
+            bicep publish $TemplateFilePath --target $publishingTargetLatest
```

Comment:
I'd recommend to not do this in here. The different versions we publish are controlled via  the `Get-ModulesToPublish` function in the pipeline. We should add 'latest' there (similiar to we nowadays publish e.g. not only `1.0.1` but also `1.0` & `1`)